Menus/PinnedToolMenu/PinnedToolButton/PinnedToolButton.cs

```diff
@@ -0,0 +1,750 @@
+﻿#if UNITY_EDITOR
```

Comment:
Agreed.  The scope has evolved into more of a ToolMenu, rather than specifically having the "pinned" design pattern.  I'll note this for a change an push in this PR.